TSOClient/FSO.UI/Framework/Font.cs

```diff
@@ -50,6 +50,11 @@ public FontEntry GetNearest(int pxSize)
                 EntryList.OrderBy(x => Math.Abs(pxSize - x.Size)).FirstOrDefault();
         }
 
+        public FontEntry GetIdeal(int pxSize)
```

Comment:
Why add&use use this when GetNearest has the same function, assuming pxSize exists within the font?

Reply:
I will re-review GetNearest, I was having unexpected behavior when using it. It seemed like based on the px it would only retrieve two of the 4(?) available sizes.

Reply:
If there's a problem with that, it might be worth finding out what's wrong there, since it's used for all other font rendering.

Reply:
Its working as intended now, when I was new to the code that was one of the first things I tried to tackle and something wasn't mixing well before, probably just me getting my feet wet with the code, my update of this PR removes a lot of excess code as the function (GetNearest) works as intended.